content/guides/concepts/cid.md

```diff
@@ -5,18 +5,39 @@ menu:
         parent: concepts
 ---
 
-A *content identifier* is a value that addresses a single piece of content in IPFS. It is mainly a cryptographic hash of the content, but is encoded as a [multihash](https://github.com/multiformats/multihash) and [multicodec](https://github.com/multiformats/multicodec). (Note: older CIDs have a different design — see [version 0](#version-0) below.)
+## Summary
```

Comment:
Would you please remove this heading? See the comment on your pinning PR for explanation: https://github.com/ipfs/docs/pull/105#discussion_r211055515